apps/datahub/src/app/record/navigation-bar/navigation-bar.component.html

```diff
@@ -1,48 +1,111 @@
-      [class]="displayMobileMenu ? 'hidden' : 'block'"
+      type="button"
+      *ngFor="let l of anchorLinks"
+      [attr.data-cy]="l.anchor"
+      gnUiAnchorLink="{{ l.anchor }}"
+      gnUiAnchorLinkDisabledClass="sm:hidden"
+      gnUiAnchorLinkInViewClass="!border-b-primary border-b-4"
+      class="font-medium hidden sm:block hover:bg-primary hover:bg-opacity-10 h-[60px] w-full px-2 items-center text-left"
       translate
     >
-      {{ activeLabel }}
+      {{ l.label }}
     </button>
-    <button
-      type="button"
-      class="h-10 inline-flex items-center justify-center p-2 rounded-md shrink-0 sm:hidden"
+    <div
+      (click)="toggleMobileMenu()"
+      class="w-full flex justify-start items-start cursor-pointer sm:hidden"
     >
-      <ng-icon class="align-middle" name="matExpandMoreOutline"></ng-icon>
-    </button>
+      <button
+        type="button"
+        class="h-[60px] inline-flex items-center justify-center p-2 rounded-md shrink-0 sm:hidden"
+      >
+        <ng-icon name="iconoirMenu" size="24px"></ng-icon>
+      </button>
+      <div
+        class="shrink truncate"
+        [class]="displayMobileMenu ? 'block' : 'hidden'"
+      >
+        <div *ngFor="let l of anchorLinks">
+          <button
+            (click)="setActiveLabel($event.target)"
+            gnUiAnchorLink="{{ l.anchor }}"
+            gnUiAnchorLinkDisabledClass="hidden"
+            class="h-[60px] block px-3 py-3 hover:bg-primary hover:bg-opacity-10 font-medium w-full text-left text-wrap"
+            translate
+          >
+            {{ l.label }}
+          </button>
+        </div>
+      </div>
+      <button
+        class="h-[60px] block px-3 py-3 rounded-md font-medium shrink text-wrap sm:hidden"
+        [class]="displayMobileMenu ? 'hidden' : 'block'"
+        translate
+      >
+        {{ activeLabel }}
+      </button>
+    </div>
+    <div
+      class="h-[60px] flex flex-row gap-3 justify-center items-center px-4 lg:pl-4 lg:justify-end"
+      [class]="displayMobileMenu ? 'border-l-2 border-gray-300' : ''"
+    >
+      <gn-ui-favorite-star
+        *ngIf="metadata?.uniqueIdentifier"
+        [record]="metadata"
+        [buttonType]="'outline'"
+        [displayCount]="false"
+        class="flex text-background content-center"
+        [style.color]="black"
+        [style.--star-toggle-enabled-color]="black"
+        [style.--star-toggle-disabled-color]="black"
+        [style.--gn-ui-button-rounded]="'8px'"
+        [style.--gn-ui-button-height]="'32px'"
+        [style.--gn-ui-button-width]="'40px'"
+        [style.--gn-ui-button-color]="black"
+        [style.--gn-ui-button-background]="'var(--color-primary-lightest)'"
+        [style.--gn-ui-button-bg-hover]="'var(--color-primary)'"
+        [style.--gn-ui-button-border]="'var(--color-gray-300)'"
+        [style.--gn-ui-button-font-size]="'24px'"
+        [style.--gn-ui-button-padding]="'4px'"
```

Comment:
FYI for later, css variables can be set on the host in the CSS file :) less verbose in HTML